<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="choose_picture">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture_show">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>256</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>180</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>180</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="res_show">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>256</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>180</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>180</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="start_histogram_show">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>256</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>180</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>180</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="res_histogram_show">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>256</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>180</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>180</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="normalization_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>直方图归一化</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
